airflow/config_templates/config.yml

```diff
@@ -2344,6 +2344,16 @@ scheduler:
       version_added: 2.0.0
       type: boolean
       default: "True"
+    parsing_pre_import_modules:
+      description: |
+        The scheduler reads dag files to extract the airflow modules that are going to be used,
+        and imports them ahead of time to avoid having to re-do it for each parsing process.
+        This flag can be set to False to disable this behavior in case an airflow module needs to be freshly
+        imported each time (at the cost of increased DAG parsing time).
+      version_added: 2.6.0
```

Comment:
I set this to 2.6 since there is still a chance it can catch the release train before it leaves the station!

Reply:
Yes it should